_docs/style/text-conventions/voice-and-tone.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Voice and tone
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Use these guidelines to write in-product text that is friendly, helpful, and inspires confidence.
+resource: false
+status: complete
+last-modified: 2018-06-01
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+The primary job of the words in a UI is to help users complete tasks with no
+confusion and minimal interruption. However, the voice and tone that we use
+with words in a UI also influence how people think and feel about Rackspace.
+
+Voice is our style, our point of view, and our personality. Tone is our
+attitude and how we are perceived by those who read (and hear) our words. Voice
+determines how we write. Tone determines our attitude to the subject and
+audience, and how we're perceived by our audience.
+
+The words we choose — and the ways we use them — should reflect our goals: building trust, inspiring confidence, making things easier, and developing a relationship with Rackspace users.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Best practices
```

Comment:
This table is really tough to read. 
Maybe the content would be more readable if we had each instruction on it's own line with a use, don't use side by side below it.

Reply:
Ok, I've worked on two other approaches to these. Hit me up today and I can show you. Thanks!